articles/native-platforms/ios-swift/06-rules.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Rules
+description: This tutorial will show you how to create a basic rule that you can use in your app.
+---
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* CocoaPods 1.0.0
+* XCode 7.3 (7D175)
+* Simulator - iPhone 6 - iOS 9.3 (13E230)
+  :::
+
+<%= include('../../_includes/_package', {
+  pkgRepo: 'native-mobile-samples',
+  pkgBranch: 'master',
+  pkgPath: 'iOS/basic-sample-swift',
+  pkgFilePath: 'iOS/basic-sample-swift/SwiftSample/Info.plist',
+  pkgType: 'replace'
+}) %>
+
+Rules are one of the cool features of Auth0. The reason behind that coolness is their flexibility, which gives you the ability to extend what Auth0 has to offer. They are just JavaScript functions which work like middlewares. To see a detailed description, please refer to [the full documentation](https://auth0.com/docs/rules).
```

Comment:
@pablolvillar We only use relative links. This `https://auth0.com/docs/rules` should be replaced with `/rules`.